jenkins/jk-setup.sh

```diff
@@ -87,10 +87,10 @@ elif [[ $COMPILER == *clang* ]]; then
     GCCcompiler=${COMPILER:6:5}
     GCCversion=${COMPILER:9:2}
     GCCversion="${GCCversion:0:1}.${GCCversion:1:1}"
-    . /afs/cern.ch/sw/lcg/contrib/gcc/${GCCversion}/${ARCH}-slc6/setup.sh
-    export PATH=/cvmfs/sft.cern.ch/lcg/contrib/llvm/latest/${ARCH}-slc6-${GCCcompiler}-opt/bin/:$PATH
+    . /afs/cern.ch/sw/lcg/contrib/gcc/${GCCversion}/${ARCH}-${LABEL}/setup.sh
+    export PATH=/cvmfs/sft.cern.ch/lcg/contrib/llvm/latest/${PLATFORM}/bin/:$PATH
```

Comment:
This is inside an `if [[ $COMPILERversion == clang_gcc* ]]; then`, or am I missing something here?